html_css/ts/app.ts

```diff
@@ -0,0 +1,75 @@
+import Paginator from "./paginator";
+import Slider from "./slider";
+import Storage from "./storage";
+import Select from "./select";
+import CustomerSlider from "./customer-slider";
+import initMobileMenu from "./mobile-menu";
+import { Form } from "./form";
+import { ReadOnly } from "./decorators/readOnly.decorator";
+import { Data } from "./models/interfaces.model";
+import { AppAbstract } from "./models/appAbstract.class";
+import { AppClasses } from "./models/enum.model";
+import './libs/my-slick';
+
+
+export default class App extends AppAbstract {
+  protected storage: Storage;
+  protected data: Data[];
+  protected BASE_URL = `https://jsonplaceholder.typicode.com/albums/`;
+
+  constructor() {
+    super();
+    this.storage = new Storage(AppClasses.Key);
+  }
+
+  @ReadOnly
+  public async init(): Promise<void> {
+    this.storage.localData = this.storage.getFromStorage(AppClasses.Key);
+    this.data = await this.setSliderData<number>(1);
+    initMobileMenu();
+    this.addStickyHeader();
+    this.initSelect();
+    new Form();
+    this.initSlider(this.data);
+    this.initPaginator();
+    this.initCustomerSlider(this.data);
+  }
+
+  public async setSliderData<T>(albumId: T): Promise<Data[]> {
+    let result: Data[]
```

Comment:
Try to avoid using of additional variable

Reply:
Done